### PR TITLE
Exchange listing changes

### PR DESCRIPTION
**Pull request by ryancreatescopy**
## Description

A series of small changes to help streamline the process for getting the information we need to successfully list exchanges with confidence.

## Related Issue

#1346

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_exchange.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+**What geographical restrictions apply to this exchange?** 
+
+<!-- If you don't know, please get in touch with the exchange. They'll likey have a list of restricted countries. -->
+
+**Which currencies are accepted by this exchange?**
+
+<!-- If you don't know, please get in touch with the exchange. They'll likey have a list of accepted currencies -->
+
+**Do you have any proof that they're an official company**
```
Comment (jjmstark) [inline .github/ISSUE_TEMPLATE/new_exchange.md:19]: I'm not sure about the wording of "official company" - I think "legitimate company" might be more what we are getting at.

File: .github/ISSUE_TEMPLATE/new_exchange.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+<!-- This is a safeguard against listing any malicious sites. You could provide a link to offical company registration. -->
```
Comment (jjmstark) [inline .github/ISSUE_TEMPLATE/new_exchange.md:21]: I would tweak wording here to say "You could provide a link to an "About" page from the exchange that provides more information about their legal entity, or a link to an official company registration". 

For many jurisdictions, there won't be a "linkable" document that attests to the corporate entity. But if there is an about page with corporate/legal info, I would expect that is OK for us.

File: .github/ISSUE_TEMPLATE/new_exchange.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+title: ""
+labels: "Type: Feature"
+assignees: ""
+---
+
+Please fill out this short form if you'd like an exchange added to ethereum.org. If it's a decentralized exchange (DEX) you'd like to list, please take a look at our [policy for listing dapps](https://ethereum.org/en/contributing/adding-products/).
+
+**What geographical restrictions apply to this exchange?** 
+
+<!-- If you don't know, please get in touch with the exchange. They'll likey have a list of restricted countries. -->
```
Comment (samajammin) [inline .github/ISSUE_TEMPLATE/new_exchange.md:13]:
```suggestion
<!-- If you don't know, please get in touch with the exchange. They'll likey have a list of restricted countries and jurisdictions. -->
```

File: .github/ISSUE_TEMPLATE/new_exchange.md (hunk 1 above)
Comment (samajammin) [reply .github/ISSUE_TEMPLATE/new_exchange.md:19]: Agreed, "legitimate" or "credible".

File: src/content/contributing/adding-exchanges/index.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+sidebar: true
+---
+
+# Adding Ethereum exchanges {#adding-ethereum-exchanges}
+
+Anyone is free to suggest new exchanges on ethereum.org.
+
+We currently list them on:
+
+- ethereum.org/get-eth
```
Comment (samajammin) [inline src/content/contributing/adding-exchanges/index.md:14]:
```suggestion
- [ethereum.org/get-eth](/en/get-eth/)
```

File: src/content/contributing/adding-exchanges/index.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+They are part of a feature that allows a user to type in where they live and see what exchanges they can use. This helps surface any geographical restrictions early.
```
Comment (samajammin) [inline src/content/contributing/adding-exchanges/index.md:16]:
```suggestion
This page allows a user to input where they live and see what exchanges they can use. This helps surface any geographical restrictions early. 
```

File: src/content/contributing/adding-exchanges/index.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+They are part of a feature that allows a user to type in where they live and see what exchanges they can use. This helps surface any geographical restrictions early. 
+
+Because of this context, we need some specific information when you suggest an exchange. 
+
+**NOTE:** If you want to list a decentralized exchange, take a look at our [policy for listing wallets and dapps](/en/contributing/adding-products/).
+
+## What we need {#what-we-need}
+
+- The geographical restrictions that apply to the exchange
+- The currencies I can use to buy ETH
```
Comment (samajammin) [inline src/content/contributing/adding-exchanges/index.md:25]:
```suggestion
- The currencies users can use to buy ETH
```

File: src/pages/get-eth.js [hunk 7]
```diff
@@ -293,11 +293,8 @@ const GetETHPage = ({ data }) => {
               All exchanges, wallets and DEXs listed on this page are not
               official endorsements, and are provided for informational purposes
               only. We add products to this page based on criteria in our{" "}
-              <Link to="/contributing/adding-products/">listing policy</Link>.
-              If we've missed a suitable product or you'd like to suggest a
-              change to the policy, let us know at{" "}
-              <Link to="mailto:website@ethereum.org">website@ethereum.org</Link>
-              .
+              <Link to="/contributing/adding-products/">dapps</Link> {" "}and <Link to="/contributing/adding-exchanges/">exchanges</Link> {" "}listing policies.
```
Comment (samajammin) [inline src/pages/get-eth.js:296]:
```suggestion
              <Link to="/en/contributing/adding-products/">dapps</Link> {" "}and <Link to="/en/contributing/adding-exchanges/">exchanges</Link> {" "}listing policies.
```

**Pull request merged**